docs/wiki/development/windows-provisioning.md

```diff
@@ -20,25 +31,20 @@ The PowerShell script `provision.ps1` is used to prepare a clean Windows 10 64 b
  * Generate the Visual Studio 2015 solution files: `cmake ..\.. -G "Visual Studio 14 2015 Win64"`
  * There should be a `OSQUERY.sln` in the build folder. Open this with Visual Studio 2015 that is already installed via the provisioning script.
 
-## Building `osqueryd.exe` and `osqueryi.exe`
+### Building `osqueryd.exe` and `osqueryi.exe`
```

Comment:
In the above two sections, this workflow is repeated a few times. This is a good opportunity to clean up it.